### ISSUE TITLE
The second most annoying bug I've ever encountered - DAs that aren't linked and have items inserted cannot be deconstructed or linked.

### ISSUE THREAD

**Issue by Probe1** [d3athrow/vgstation13#4561]
If the RnD console is somehow disassembled all the machines that were previously linked to it must be deconstructed and reconstructed or else they cannot be linked again.

That in itself is an annoying as fuck bug.

But if the Destructive Analyzer just so happens to be unlinked and just so happens to have an item in it, it cannot be deconstructed without blowing it up with a bomb.  It won't let you screwdriver it open because an item is inside.  It won't let you re-link it because it was linked to the previous console that was deconstructed.  So you got yourself an immovable object.  

The big question is why the fuck can't you just take it apart when an item is inside?  All it has to do is spit the item out when deconstructed.  Shit.  That made me peeved..

**Comment by dylanstrategie** [comment]
I'm fairly sure this also happens with the cloning apparatus

**Comment by 9600bauds** [comment]
While we're at it I would also like to add that you need access to eject an item, but not to put it in. Which is ridiculous really

**Comment by Probe1** [comment]
With the cloner you can just toggle the apc breaker to eject someone.  With the DA you have a goddamn perfect wall that can't be removed.

**Comment by FudgePucker** [comment]
You know what, what the fuck is so special with the DA that you can't just reach in and take out the item that you inserted?
Can't we just add an eject item verb to it?

**Comment by CptWad** [comment]
Also to be added to the problem: Try deconning the DA over and over again
Results in infinite wired machine frames

**Comment by Exxion** [comment]
Out of curiosity, what's the most annoying bug you've ever encountered?

**Comment by Probe1** [comment]
@Exxion 
https://github.com/d3athrow/vgstation13/pull/4236

I don't understand the process.  I know he called it double pooling.  I only understand what was happening.  The singlo was released by an antag but I managed to blow it and the Captain wanted us to stay despite engineering and every engine sans the TEG being gone.  So we ordered a new AME and set it up where the singlo and SMES used to be.

There was some kind of double pooling bug.  When we placed down cable coils, from every third to every fifth cable placed another already placed wire would delete itself, all by its own.  So for every two cable lengths you placed, another random one nearby would delete.  

I wired that fucking AME.  It took me 10+ cable coils to do about 15 tiles of wiring it but I wired that fucking thing and the new SMES I built.
